charts/pgbouncer/templates/pgbouncer-deployment.yaml

```diff
@@ -55,11 +55,10 @@ spec:
       serviceAccountName: {{ template "pgbouncer.serviceAccountName" . }}
       containers:
         - name: pgbouncer
-          image: {{ template "pgbouncer.image" . }}
+          image: {{ include "pgbouncer.image" . | trim }}
           securityContext: {{ toYaml .Values.securityContext| nindent 12 }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-            {{- include "pgbouncer_environment" . | indent 12 }}
```

Comment:
we have to also remove 
https://github.com/astronomer/astronomer/blob/68f7c8c03a1487af0d7de02c914c4b1f5e3cdcaf/charts/pgbouncer/templates/_helpers.yaml#L43